setup/README.md

```diff
@@ -21,7 +21,7 @@ We assume you have a running Kubernetes cluster already. Since this tutorial wor
 
 > **MicroK8s**: DNS addon must be activated using `sudo microk8s enable dns`.
 
-> This tutorial was tested on a machine running Ubuntu 20.04. It worked an Azure Kubernetes Service (AKS) cluster, an Elastic Kubernetes Service (EKS) cluster based on EC2 instances and with a local minikube cluster.
+> This tutorial was tested on a machine running Ubuntu 20.04. It worked with an Azure Kubernetes Service (AKS) cluster, an Elastic Kubernetes Service (EKS) cluster based on EC2 instances, a Google Kubernetes Engine (GKE) cluster and with local minikube and MicroK8s clusters.
```

Comment:
don't think it needs this except for the OS version. Think the rest is given above...